### PR TITLE
addressing comments from last code review

### PR DESCRIPTION
**Pull request by krachwal**
Created a separate PR since [the last PR ](https://github.com/gt-sse-center/ramanujan-machine-web/pull/44)was very large and I wanted to keep this easy for anyone to review.

### CODE REVIEW

File: python-backend/wolfram_client.py [hunk 1]
```diff
@@ -41,6 +42,7 @@ def ask(query: str, include_pod: str = None) -> dict:
         else:
             try:
                 result = json.loads(result.content)
+                logger.debug(result)
```
Comment (krachwal) [inline python-backend/wolfram_client.py:45]: moved here from 67 below

File: python-backend/wolfram_client.py [hunk 2]
```diff
@@ -60,11 +61,15 @@ def closed_form(expression: str) -> dict:
...
+            assert len(expression) == 200
+        except AssertionError:
+            logger.warning("Truncating decimal value to 200 characters to keep within Wolfram API query limit")
+
+        try:
             result = WolframClient.ask(query=expression[:200], include_pod="PossibleClosedForm")
             # only want to return: queryresult -> pods[0] -> subpods
             # infos has metadata on the subpods e.g. the constant names and links to reference content
             # Note: the index of the infos does not line up with the index of the closed form results
-            logger.debug(result)
```
Comment (krachwal) [inline python-backend/wolfram_client.py:67]: just moving it

File: python-backend/wolfram_client.py [hunk 3]
```diff
@@ -60,11 +61,15 @@ def closed_form(expression: str) -> dict:
         """
         try:
             # Wolfram has a 200 character input limit
+            assert len(expression) == 200
```
Comment (krachwal) [inline python-backend/wolfram_client.py:64]: need to log this scenario for debugging purposes

File: react-frontend/vite.config.js [hunk 4]
```diff
@@ -11,12 +11,6 @@ export default defineConfig(() => {
 			outDir: 'build'
 		},
 		base: '/form/',
-		server: {
```
Comment (krachwal) [inline react-frontend/vite.config.js:14]: 80 is default - do not need to state it explicitly

File: python-backend/wolfram_client.py (hunk 3 above)
Comment (varun646) [reply python-backend/wolfram_client.py:64]: Should this be <= ? Just going off the comments from the last PR

Comment (krachwal) [reply python-backend/wolfram_client.py:64]: 🤦🏻 multitasking is bad

**Review by varun646 — APPROVED** [review]

**Pull request merged**